src/server/js/resources/rooms/schema.js

```diff
@@ -12,13 +12,25 @@ const typeDefs = `
         getRooms: [Room],
         roomsByHouse(houseId: ID!): [Room],
     }
+    
+    extend type Mutation {
+        createRoom(capacity: Int, description: String, house: String): String,
+        updateRoom(id: String, capacity: Int, description: String, house: String): Room,
+        deleteRoom(id: String): Room,
+    }
 `;
 
 const resolvers = {
     Query: {
         getRooms: (root, args, context, info) => RoomModel.getAll(),
         roomsByHouse: (root, args, context, info) => RoomModel.getAllRoomsByHouse(args.houseId),
     },
+
+    Mutation: {
```

Comment:
we should check that the person performing create/update/delete operations is an administrator for the house being managed!